src/components/Api.js

```diff
@@ -0,0 +1,90 @@
+class Api {
+  constructor(options) {
+    // constructor body
+    this._baseUrl = options.baseUrl;
+    this._headers = options.headers;
+    this._spinner = document.querySelector(".spinner");
```

Comment:
`    this._spinner = document.querySelector(".spinner");
`

This is not needed in `Api` (and not used)

Reply:
yes, agree. I was just trying to use case from theory lessons and didn't clear all code, sorry.